sdk/python/feast/infra/offline_stores/file.py

```diff
@@ -210,6 +213,89 @@ def evaluate_historical_retrieval():
         job = FileRetrievalJob(evaluation_function=evaluate_historical_retrieval)
         return job
 
+    @staticmethod
+    def get_historical_features_by_view(
+        config: RepoConfig,
+        feature_views: List[FeatureView],
+        feature_refs: List[str],
+        entity_df: Union[pd.DataFrame, str],
+        registry: Registry,
+        project: str,
+        start_date: Optional[datetime],
+        end_date: Optional[datetime],
+        full_feature_names: bool,
+    ):
+        if not (isinstance(entity_df, pd.DataFrame) or isinstance(entity_df, str)):
+            raise ValueError(
+                f"Please provide an entity_df of type {type(pd.DataFrame)} instead of type {type(entity_df)}"
```

Comment:
Should there be two exception messages here?

Reply:
thanks!